documentation/ci-fix.md

```diff
@@ -49,18 +49,30 @@ Or you can run it in [OpenAPI Hub](https://portal.azure-devex-tools.com/tools/st
 Refer to [Semantic and Model Violations Reference](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/Semantic-and-Model-Violations-Reference.md) for detailed description of validations and how-to-fix guidance.
 
 ## Breaking Change Check
-
+There are two kind of breaking change checks: same version breaking change check and cross version breaking change check. Both are using the 'oad' tool to compare the difference between two swaggers.
+- The same version breaking change check is for checking the existing swagger, it compares the changed swaggers to the swaggers before the change and ensure the change don't break the current version.
+- The cross version breaking change check is to find a last version swagger as a base swagger , compare the base swagger and the current swagger. Make sure the change don't break the following older versions:
+1. last stable version swagger which exists less than 3 year.
```

Comment:
1. The latest stable version which released within 3 years.
2. When there is no stable version released within 3 years, then the latest preview version which released within 1 years.